ql/src/test/results/clientpositive/llap/lateral_view_ppd.q.out

```diff
@@ -316,29 +332,36 @@ STAGE PLANS:
           filterExpr: (key = '0') (type: boolean)
           Filter Operator
             predicate: (key = '0') (type: boolean)
-            Lateral View Forward
-              Select Operator
-                expressions: value (type: string)
-                outputColumnNames: value
-                Lateral View Join Operator
-                  outputColumnNames: _col1, _col6
-                  Select Operator
-                    expressions: _col1 (type: string), _col6 (type: int)
-                    outputColumnNames: _col0, _col1
-                    ListSink
-              Select Operator
-                expressions: array(1,2,3) (type: array<int>)
-                outputColumnNames: _col0
-                UDTF Operator
-                  function name: explode
-                  Filter Operator
-                    predicate: (col > 1) (type: boolean)
```

Comment:
This, like others, happened because we are going through the CBO path.

In this case, the optimizer for the planner changed.  There was code missing in OpProcFactory (more details in a different comment)